SCIF app Inspect failure after merge 5136 / 5137 to master

After merging the inspect metadata PRs #5136 #5137 to master I see a failure:

```
                --- FAIL: TestE2E/PAR/INSPECT/inspect_command/SIF/list_apps (0.14s)
```

The PRs _did_ pass - but this needs investigation.

https://circleci.com/api/v1.1/project/github/hpcng/singularity/52828/output/108/0?file=true&allocation-id=5f89acdc7655844a5c12e877-0-build%2F40E08C86

--- a/cmd/internal/cli/inspect.go
+++ b/cmd/internal/cli/inspect.go
@@ -198,8 +198,8 @@ func newCommand(allData bool, appName string, img *image.Image) *command {
 		if err == nil {
 			sylog.Debugf("Using %s SIF descriptor", metadataJSON)
 			command.sifMetadata = metadata
-			if allData {
-				// copy app attributes as they are not copied when --all is selected
+			if listApps || allData {
+				// copy app attributes for related flags as they are not copied by default
 				command.metadata.Attributes.Apps = metadata.Attributes.Apps
 			}
 		} else if err != image.ErrNoSection {

